packages/next/client/image.tsx

```diff
@@ -127,13 +128,37 @@ export type ImageProps = Omit<
   onLoadingComplete?: OnLoadingComplete
 }
 
+type ImageElementProps = Omit<
+  JSX.IntrinsicElements['img'],
+  'src' | 'srcSet' | 'ref' | 'width' | 'height' | 'loading'
+> & {
+  raw: boolean
```

Comment:
Lets just pass `layout` instead of a boolean

Reply:
done